includes/wsuwp-shortcode-fundselector-utils.js

```diff
@@ -1,5 +1,7 @@
 // Does our namespace exist
 window.wsuwpUtils = window.wsuwpUtils || {};
+var maximumGift = 100000;
```

Comment:
Since these are global, use the standard convention of uppercase: `MAXIMUM_GIFT`